product_supplierinfo_revision/tests/test_product_supplierinfo_revision.py

```diff
@@ -16,6 +16,7 @@ def setUpClass(cls):
         cls.vendor = cls.env['res.partner'].create({
             'name': 'Suplier test',
             'supplier': True,
+            'picking_warn': 'no-message',
```

Comment:
Instead of this, put `at_install = False` and `post_install = True` for executing tests at the end